CondCore/CondDB/test/condTestRegression.py

```diff
@@ -137,10 +145,12 @@ def run(self, rel, arch, readOrWrite, dbName):
           cmsPath = out[:ind-1]
           # using wildcard to support the path for normal ( BASE/ARCH/cms/cmssw/RELEASE ) and patch releases ( BASE-PATCH/ARCH/cms/cmssw-patch/RELEASE )
           releaseDir = '%s/%s/cms/*/%s' %(cmsPath,arch,rel)
-
+          
+          payloadScript = 'test/%s/testReadWritePayloads' % arch
+              
           cmd =  'source %s/cmsset_default.sh; export SCRAM_ARCH=%s; cd %s/src ; eval `scram runtime -sh`; cd - ; ' %(cmsPath,arch,releaseDir)
           cmd += "echo 'CMSSW_BASE='$CMSSW_BASE; echo 'RELEASE_BASE='$RELEASE_BASE; echo 'PATH='$PATH; echo 'LD_LIBRARY_PATH='$LD_LIBRARY_PATH;"
-          cmd += '$LOCALRT/test/%s/testReadWritePayloads %s sqlite_file:///%s/%s ' % (arch, readOrWrite, self.dbDir, dbName)
```

Comment:
@ggovi , here you are setting env for releaseDir (an old release build with old version of boost) and then running $LOCALRT/test/%s/testReadWritePayloads from local dev area ( which is linked against new boost version 1.63).